acceptancetests/assess_cross_model_relations.py

```diff
@@ -70,13 +75,9 @@ def assess_cross_model_relations_multiple_controllers(args):
     first_bs_manager = BootstrapManager.from_args(args)
     with first_bs_manager.booted_context(args.upload_tools):
         first_model = first_bs_manager.client
-        # Need to share juju_data/home
-        second_bs_manager.client.env.juju_home = first_model.env.juju_home
-        with second_bs_manager.existing_booted_context(
-                second_args.upload_tools):
+        with second_bs_manager.booted_context(second_args.upload_tools):
```

Comment:
Do primary and secondary correspond to offerer and consumer? If so, let's try and stick to the same verbiage inside the test.